sql/mf_iocache_encr.cc

```diff
@@ -182,6 +185,10 @@ static int my_b_encr_write(IO_CACHE *info, const uchar *Buffer, size_t Count) {
     if (!Count) DBUG_RETURN(0);
   }
 
+#ifndef DBUG_OFF
```

Comment:
Is the Valgrind issue present in 5.7 too?

Reply:
yes